WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/DevicePortal.cs

```diff
@@ -175,7 +175,7 @@ public string PlatformName
             }
         }
 
-        public HoloLensDevicePortal HoloLens;
+        public MixedRealityPortal MixedReality = new MixedRealityPortal(this);
```

Comment:
Based in this pattern, are ALL sub-portal going to be initialized by default? If so, they should be marked as readonly, so client code does not modify or (worse) invalidate the instances.